content/hi/docs/concepts/architecture/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: क्लस्टर आर्किटेक्चर
+weight: 30
+description: >
+  कुबेरनेट्स के पीछे वास्तु अवधारणाएँ।
+---
+
```

Comment:
It seems that the localization of this file may be incomplete. There is a diagram included in the `{{ <figure> }} ` shortcode in the English file _[(check it out on Line 8)](https://github.com/kubernetes/website/blob/17f080049278a691d417c44decddbdb297f744b5/content/en/docs/concepts/architecture/_index.md?plain=1)_ that should be incorporated into this Hindi file here.